testing/lolcat/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Contributor: Daniel Isaksen <d@duniel.no>
+# Maintainer: Simon Frankenberger <simon-alpine@fraho.eu>
+pkgname=lolcat
+pkgver=1.0
+pkgrel=0
+pkgdesc="High-performance implementation of lolcat" 
+url="https://github.com/jaseg/lolcat/" 
+arch="all" 
+license="WTFPL" 
+depends="" 
+makedepends="" 
+install="" 
+source="$pkgname-$pkgver.tar.gz::https://github.com/jaseg/${pkgname}/archive/v$pkgver.tar.gz
+fix-url-and-version.patch" 
+
+build() {
+        make
+}
+
+check() {
+        ./lolcat --help 2>/dev/null
```

Comment:
We do not accept this kind of testing. So please remove check() and add `options="!check" # no test suite`